e2e/cypress/support/api/system.js

```diff
@@ -233,12 +228,6 @@ Cypress.Commands.add('shouldNotRunOnCloudEdition', () => {
 });
 
 function isTeamEdition() {
-    const isTeamServer = Cypress.env('serverEdition') === Constants.ServerEdition.TEAM;
-
-    if (isTeamServer) {
-        return cy.wrap(true);
-    }
-
```

Comment:
This check for team and the other for cloud were removed to rely server edition on license loaded in the server, and not via Cypress env variables.